src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessThreadTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Security;
+using System.IO;
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessThreadTests : ProcessTestBase
+    {
+        [Fact]
+        public void TestCommonPriorityAndTimeProperties()
+        {
+            ProcessThreadCollection threadCollection = _process.Threads;
+            Assert.True(threadCollection.Count > 0);
+
+            ProcessThread thread = threadCollection[0];
+
+            if (ThreadState.Terminated != thread.ThreadState)
+            {
+                Assert.Equal(_process.BasePriority, thread.BasePriority);
+                Assert.True(thread.CurrentPriority >= 0, "Thread CurrentPriority");
+                Assert.True(thread.PrivilegedProcessorTime.TotalSeconds >= 0, "Thread PrivilegedProcessorTime");
+                Assert.True(thread.UserProcessorTime.TotalSeconds >= 0, "Thread UserProcessorTime");
+                Assert.True(thread.TotalProcessorTime.TotalSeconds >= 0, "Thread TotalProcessorTime");
```

Comment:
Is there any test we could do, outer loop if necessary, that spins a thread doing a bunch of computation, and then we could verify that its total processing time is higher than some expected value, rather than just >= 0?